pom.xml

```diff
@@ -179,6 +179,46 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
`spring-boot-starter-web` already includes Jackson as a JSON de-serializer, is GSON necessary?

Reply:
This dependency is requested by the auto generated classes